hooks.py

```diff
@@ -41,12 +41,19 @@ def companies_updated(companies):
 
 def notices_updated(notices):
     for notice in notices:
+        if " : PLACEMENT" in notice['text']:
```

Comment:
It's better to match on the exact string (ie `Type : PLACEMENT`). Also need to handle the case where both internship and placement match.

Reply:
Yeah, match the complete string. Also, you need to think about cases where the string might be in the notice but that might not be in the particular type. Might not be very prevalent or common, definitely think about it though.